### PR TITLE
feat: Export the start block number and slice the light client log.

### PR DESCRIPTION
**Pull request by yanguoyu**
1. Export the start block number https://github.com/Magickbase/neuron-public-issues/issues/386
2. Slice the export light client log https://github.com/Magickbase/neuron-public-issues/issues/378

### CODE REVIEW

File: packages/neuron-wallet/src/controllers/export-debug.ts [hunk 1]
```diff
@@ -95,30 +97,31 @@ export default class ExportDebugController {
         release,
         vcredist,
       },
+      wallets: wallets.map(v => ({
+        id: v.id,
+        name: v.name,
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/controllers/export-debug.ts:102]: `Name` of a wallet is kind of private and it's better not to be exposed in public, and `id` is sensitive too, how about masking some chars of the id in the log

Comment (yanguoyu) [reply packages/neuron-wallet/src/controllers/export-debug.ts:102]: I will remove the name field. Id is a random string created by `uuid`, I don't know whether id is sensitive

Comment (yanguoyu) [reply packages/neuron-wallet/src/controllers/export-debug.ts]: I have removed the `walletId` in `hd_public_key_info.csv` and used `index` to recognize it as one wallet, and also match the start block number that the user sets.

**Review by Keith-CY — APPROVED** [review]

**Comment by Danie0918** [comment]
@homura @devchenyan Please have a review.

**Review by devchenyan — APPROVED** [review]

**Pull request merged**